attributes/remove_plugins.rb

```diff
@@ -0,0 +1,9 @@
+# Plugins on this list will be removed on each chef run.
+# Adding plugins to this list requires a minor version bump.
+# Removing plugins from this list requires a major version bump.
+default['ros_buildfarm']['jenkins']['remove_plugins'] = %w[
+analysis-core
+config-file-provider
+cvs
+icon-shim
```

Comment:
It would be nice to match the indentation used in `plugins.rb`. I understand that it isn't the same data type, though.
```suggestion
  analysis-core
  config-file-provider
  cvs
  icon-shim
```

Reply:
It still makes sense to do a visual indent within the array though. Good call :+1: